model/RdsResultStorage.php

```diff
@@ -189,81 +136,48 @@ public function storeItemVariables(
                 $callIdItem
             );
         }
-        
+
         $this->getPersistence()->insertMultiple(self::VARIABLES_TABLENAME, $dataToInsert);
     }
 
-    /*
-     * retrieve specific parameters from the resultserver to configure the storage
-     */
-    public function configure($callOptions = array())
+    public function storeRelatedTestTaker($deliveryResultIdentifier, $testTakerIdentifier)
```

Comment:
Why did you remove the docblock? (same about some other methods)

Reply:
Most of them were just replicating the interfaces' ones. I ran through all of them to check when it was necessary...